### DISCUSSION TITLE
Scanning Frequencies

### DISCUSSION THREAD

**Discussion by ritts3775** [DSheirer/sdrtrunk#1807] [Q&A]
I've recently started with SDRtrunk. I was able to get it working although I am running another software alongside of it for a different application but it isn't showing some of the frequencies that is being show on the other program. I am not sure what I am doing wrong. 

I am using bucks county p2 system. It hits all of the frequencies that it shown on radio reference but there are 2 others that aren't shown in RR 770.556250 and 769.506250 These are the ones I am loosing transmissions on. 

I am only using one RTL-SDR.

https://www.radioreference.com/db/sid/7699

**Comment by JustApu** [comment]
@ritts3775 I'm seeing both of those frequencies on the RadioReference page you linked. That said, as long as you have all the control channels in your configuration, it should not matter that you are missing non-control channels.

``I am only using one RTL-SDR.``

This is more likely the problem. The RTL-SDR dongles have a maximum bandwidth of 2.4 MHz. If it is tuned to the 772.98125 MHz control channel, it can't also monitor anything less than 770.58125 MHz. One of the other possible control channels, the lower limit is going to be an even higher frequency.

**Reply by ritts3775** [reply]
Okay- I've ordered another dongle due to some videos that I watched. It will be here tomorrow. Hopefully this resolves the issue. Thank you for the help!

**Reply by ritts3775** [reply]
Now as you see there are is the control channel and then there is 3 alternatives. Do I need to program the alternatives in there as well?

**Reply by JustApu** [reply]
You don't absolutely need the other control channels programmed if the system is just using the one. But, if the system switches control channels (for maintenance, radio failure, etc.) and you don't have the "new" control channel programmed in, you will miss transmissions. (They won't switch to a random channel; they use the designated alternate channels.) There's no harm in having the alternates programmed in; once you lock on to the active control channel, the software listens to the commands there to determine what other channels it needs to tune to. The "extra" channels won't take up a dongle or anything unless they are in use.

``I've ordered another dongle``

To @Ctpolicelive's point, you'll may need three RTL-SDRs (or some other SDR with a wider bandwidth) to fully cover the system. Remember, each will only cover 2.4 MHz of spectrum. It depends on how busy the system is, which control channel is being used and if the alternate control channels are used for voice transmissions. If the channels above 772.98125 MHz are not being used, two dongles should work. If the 773 & 774 MHz channels are in use, you're going to need the third dongle to get 100% coverage. But one more is going to be a good start.

**Reply by ritts3775** [reply]
I'll have to look back at the call logs to see if they are even being used. I don't think they are due to them being control channels only.

**Reply by ritts3775** [reply]
This seems to be working a lot better with the additional SDR. Thank you for your help!

**Comment by ghost** [comment]
> I am only using one RTL-SDR.

Buy more SDR's, you most likely need at least 2 or 3.

**Reply by ritts3775** [reply]
Okay- I've ordered another dongle due to some videos that I watched. It will be here tomorrow. Hopefully this resolves the issue. Thank you for the help!

**Reply by ritts3775** [reply]
This seems to be working a lot better with the additional SDR. Thank you for your help!
